starknet_py/tests/e2e/manifest-path.template

```diff
@@ -0,0 +1 @@
+path_to_cairo1_compiler/Cargo.toml
```

Comment:
Only have one question. Where exactly should it be located? and what is it for? Let's maybe add a description here and above `get_compiler_manifest` fixture